### ISSUE TITLE
error when i installed the update via the submod updater plugin, i use macOS

### ISSUE THREAD

**Issue by PanickingPanini** [tw4449-s-MAS-Submods/tw4449-Custom-Room-Selection-Pack-Main-Repository#51]
[code]
I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/game/Submods/Submod Updater Plugin/submod_updater_plugin.rpy", line 3, in script
    init -990 python:
  File "game/game/Submods/Submod Updater Plugin/submod_updater_plugin.rpy", line 12, in <module>
    settings_pane="sup_setting_pane"
  File "game/zz_submods.rpy", line 118, in __init__
SubmodError: A submod with name 'Submod Updater Plugin' has been installed twice. Please, uninstall the duplicate.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/game/Submods/Submod Updater Plugin/submod_updater_plugin.rpy", line 3, in script
    init -990 python:
  File "/Applications/DDLC.app/Contents/Resources/autorun/renpy/ast.py", line 814, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "/Applications/DDLC.app/Contents/Resources/autorun/renpy/python.py", line 1719, in py_exec_bytecode
    exec bytecode in globals, locals
  File "game/game/Submods/Submod Updater Plugin/submod_updater_plugin.rpy", line 12, in <module>
    settings_pane="sup_setting_pane"
  File "game/zz_submods.rpy", line 118, in __init__
SubmodError: A submod with name 'Submod Updater Plugin' has been installed twice. Please, uninstall the duplicate.

Darwin-18.7.0-x86_64-i386-64bit
Ren'Py 6.99.12.4.2187
Monika After Story 0.12.12
[/code]

**Comment by PanickingPanini** [comment]
i know that its a duplicate of smth but idk where to find it

**Comment by Cdino11** [comment]
its a duplicate game folder in your game folder

**Comment by PanickingPanini** [comment]
oh
ill try to delete that rn thanks

**Comment by PanickingPanini** [comment]
it worked, much thanks :)
